templates/style/_navbar.scss

```diff
@@ -25,6 +25,9 @@ div.nav-container {
         }
         &:after {
             font-size: 0.8em;
+            position: absolute;
```

Comment:
This makes the menu feel very cramped to me, and especially there is not enough padding on the right-hand side of the arrow.

Reply:
Do you have a screenshot by any chance?

Reply:
Current look:

![image](https://user-images.githubusercontent.com/81079/97458968-8af4dc80-193b-11eb-9b5e-2b1a3b97b0b6.png)

On this branch:

![image](https://user-images.githubusercontent.com/81079/97458999-921bea80-193b-11eb-8e14-b1ea6f7200af.png)

Reply:
Yes, that's exactly how I intended to have it. Do you prefer having the arrow right next to the text maybe?

Reply:
(that screenshot also shows off the extra whitespace after short crate names that I dislike).

Reply:
IMO the whitespace from the left-border to the icon/text and from the drop-down-arrow to the right border should be the same to make it feel balanced.

Reply:
If you have a solution which allows to have a way to have just a max-width and that the items will remove the display of the text by themselves, I'd be glad to hear it (a lot of CSS remains unknown to me).

Reply:
Changing `width: 200px;` to `max-width: 200px;` appears to work to me.

Reply:
It "works". However, the items on the right won't remove the text at the "right" time when the windows size is changing on smaller crate titles.

Reply:
You mean that there's extra whitespace available they could use when they shrink? Doesn't seem like a problem to me, the exact amount of whitespace there will be influenced by things like the font selected anyway, and not many people drag window sizes around and notice if the responsiveness is _exactly_ right. As long as the media queries target the maximum that could be used it seems fine.

Reply:
I guess this is good enough. I'll do that then!